### DISCUSSION TITLE
Where reside urls in database?

### DISCUSSION THREAD

**Discussion by Tychodewaard** [vanjarosoftware/Vanjaro.Platform#1375] [Q&A] [answered]
I like to change the 200 and 301 of a page. But if this somehow screws things up, where can I correct this in the database?

**Comment by hshimanshusharma** [comment, marked as answer]
> I like to change the 200 and 301 of a page. But if this somehow screws things up, where can I correct this in the database?

You can see the TabUrls table 
SELECT * from TabUrls
